app/components/Modals/SendModal/ConfirmDetails/ConfirmDetails.jsx

```diff
@@ -14,7 +15,7 @@ type Props = {
 
 export default class ConfirmDetails extends React.Component<Props> {
   getRecipientData = (key: string) =>
-    this.props.recipientData[key] || 'Not specified'
+    get(this.props.recipientData, key) || 'Not specified'
```

Comment:
Just to call this out before it is commented on - I'm aware the third argument in `get` acts as a default, but this only works for `undefined` values _not_ `null`. In this case, the function that is responsible for generating `recipientData` returns `null`'s for empty keys (the `.get` method on the native implementation of `URL` is the source of this). Since this doesn't cause a problem anywhere else I thought it'd be cleaner to write it this way rather than having to convert them to `undefined`'s in `parseQrCodeData`.